nut/DOCS.md

```diff
@@ -13,10 +13,11 @@ are [compatible][nut-compatible].
 The Network UPS Tools (NUT) project is the combined effort of
 many [individuals and companies][nut-acknowledgements].
 
-Be sure to add the NUT integration after starting the add-on.
+After starting this add-on, the Home Assistant NUT core integration must also be added and configured. 
 
-**Note**: _The host `a0d7b954-nut` can be used to allow Home Assistant to
-communicate directly with the addon_
+> **Note**: _The host `a0d7b954-nut` can be used to allow Home Assistant to
+communicate directly with the add-on_.  Specifically, when starting the core integration for the first time the 'host' value will default to _localhost_, 
+which will not work, the value `a0d7b954-nut` should be supplied for the core integration to communicate with this add-on. 
```

Comment:
This is not a quotation.

```suggestion
**Note**: _The host `a0d7b954-nut` can be used to allow Home Assistant to
communicate directly with the add-on_. Specifically, when starting the core integration for the first time the 'host' value will default to _localhost_, 
which will not work, the value `a0d7b954-nut` should be supplied for the core integration to communicate with this add-on. 
```

Reply:
I already clearly states the host to use, why list what not to use?
Maybe add '192.168.1.1` doesn't work either?

Reply:
Specifically because the NUT core integration supplies that value as it's default.  It's VERY difficult to get this integration started for the first time.  I spent hours working thru issues with both the add-on and integration yesterday and am simply trying to find a way to make this process simpler for the next person. I'm done tho, I can't suggest anything else.  Close this if you like, I know how it works now, the next person doesn't tho.

Reply:
Ok closing. Thanks for willing to contribute though 👍